MidasCivil_Adapter/CRUD/Create/Loads/AreaDifferentialTemperatureLoad.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Structure.Loads;
+using BH.oM.Structure.Elements;
+using BH.oM.Geometry;
+using System.Collections.Generic;
+using System.IO;
+using BH.Engine.Adapter;
+using BH.oM.Adapters.MidasCivil;
+
+namespace BH.Adapter.MidasCivil
+{
+    public partial class MidasCivilAdapter
+    {
+        public bool CreateCollection(IEnumerable<AreaDifferentialTemperatureLoad> areaDifferentialTemperatureLoads)
+        {
+            string loadGroupPath = CreateSectionFile("LOAD-GROUP");
+
+            foreach (AreaDifferentialTemperatureLoad areaDifferentialTemperatureLoad in areaDifferentialTemperatureLoads)
+            {
```

Comment:
You need a check here to say that `AreaDifferentialTemperature` with a top and bottom temperature are being accepted. You can either:
1. Accept `AreaDifferentialTemperature` loads with mulitple positions/temperatures and ignore that information and raise a warning
2. Don't accept `AreaDiffernetialTemperature` loads that have more than 2 positions/temperatures
3. You should also make sure the positions defined at 0 and 1.0 as they are all that is supported

You also need to capture whether any of the objects being pushed do not have a SectionProperty, it can still be pushed (as we are not reliant on absolute distances) but a warning should be raised as the load will not parse in MidasCivil correctly.